docs/what_is_data_science/How you can learn Data Science.md

```diff
@@ -0,0 +1,14 @@
+# How you can learn Data Science
+
+Data Science isn't just for Data Scientists! As a profession, we're passionate about sharing these skills and techniques.
+
+The [NHS Data Science community](https://data-science-community.analystx.uk/) is the home of spreading data science knowledge within the NHS.
+
+You can also learn lot about data science by simply getting to know the wider cross-govt/health community:
+
+- [Govt Data Science Community](https://www.gov.uk/service-manual/communities/data-science-community)
+- [NHS R Community](https://nhsrcommunity.com/)
+- [NHS.Pycom](https://nhs-pycom.net/)
+- [AnalystX](https://analystx.uk/)
+
+These have some overlap, but each has it's own niche, depending if you want to focus more on Python, R, etc.
```

Comment:
feels weird to end a page with the word etc so maybe change this to: 

These have some overlap, but each has it's own niche, depending on what language or area you want to focus more on (e.g. Python, R, specific use cases).

Reply:
Changed,